wormhole-connect/src/views/v2/Bridge/index.tsx

```diff
@@ -136,10 +136,10 @@ const Bridge = () => {
 
     const routeState = routeStates?.find((rs) => rs.name === route);
 
-    if (routeState?.available && routeState?.supported && !selectedRoute) {
+    if (routeState?.supported && routeState?.available) {
```

Comment:
We don't need to check selectedRoute as the side-effect will no longer triggered its changes.